Source/ScalingHeaderScrollView.swift

```diff
@@ -124,6 +124,9 @@ public struct ScalingHeaderScrollView<Header: View, Content: View>: View {
         didPullToRefresh != nil
     }
 
+    /// Scroll view did reach bottom
+    private var didReachBottom: (() -> Void)?
```

Comment:
why is this between 2 computed Bool properties? Please place to other closures. The order of these parameters declaration should be the same as order of modifers declaration